src/main/java/juja/sqlcmd/Application.java

```diff
@@ -1,7 +1,190 @@
+        printAffectedRows(affectedRows);
+    }
+
+    private void changePasswordInUserTable(String name, String password) {
+        String sqlQuery = String.format("UPDATE " + USER_TABLE_NAME + " SET password = '%s'" + " WHERE name='%s'", password, name);
+        printStatus("Updating", USER_TABLE_NAME);
+        int affectedRows = executeUpdateQuery(sqlQuery);
+        printAffectedRows(affectedRows);
+    }
+
+    private void removeFromUserTable(String name) {
+        String sqlQuery = String.format("DELETE FROM " + USER_TABLE_NAME + " WHERE name='%s'", name);
+        int affectedRows = executeUpdateQuery(sqlQuery);
+        printAffectedRows(affectedRows);
+    }
+
+    private void printTable(String tableName) {
+        String sqlQuery = String.format("SELECT * FROM %s order by %<s.id", USER_TABLE_NAME);
+        try (PreparedStatement statement = connection.prepareStatement(sqlQuery)) {
+            printStatus("Printing", tableName);
+            statement.execute();
+            ResultSet resultSet = statement.getResultSet();
+
+            if (!resultSet.isBeforeFirst()) {
+                System.out.println("table is empty" + LINE_SEPARATOR);
+                return;
+            }
+
+            StringBuilder stringBuilder = new StringBuilder();
+            int columnsCount = getColumnsCountFrom(resultSet);
+            while (resultSet.next()) {
+                String columnsSeparator = " |";
+                for (int i = 1; i <= columnsCount; i++) {
+                    stringBuilder.append(resultSet.getString(i));
+                    stringBuilder.append(columnsSeparator);
+                }
+                stringBuilder.delete(stringBuilder.length() - columnsSeparator.length(), stringBuilder.length());
+                stringBuilder.append(LINE_SEPARATOR);
+            }
+            System.out.println(stringBuilder);
+            resultSet.close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private void insertIntoUserTable(String name, String password) {
+        String sqlQuery = String.format("INSERT INTO " + USER_TABLE_NAME + "(name, password) VALUES('%s','%s') ", name, password);
+        printStatus("Inserting into", USER_TABLE_NAME);
+        int affectedRows = executeUpdateQuery(sqlQuery);
+        printAffectedRows(affectedRows);
+    }
+
+    private void printAffectedRows(int affectedRows) {
+        if (affectedRows < 0) {
+            System.out.println("The query ended unsuccessfully, 0 rows affected" + LINE_SEPARATOR);
+        } else {
+            System.out.println("The query ended successfully, " + affectedRows + " row(s) affected" + LINE_SEPARATOR);
+        }
+    }
+
+    private int executeUpdateQuery(String sqlQuery) {
+        try (PreparedStatement statement = connection.prepareStatement(sqlQuery)) {
+            return statement.executeUpdate();
+        } catch (SQLException e) {
+            // gotcha
```

Comment:
It's bad style. 
Why you catch exception and do nothing.
You should catch it in high level, for example in your main method